R/Connection.R

```diff
@@ -494,45 +494,57 @@ setMethod(
   "dbQuoteIdentifier", c("OdbcConnection", "SQL"),
   getMethod("dbQuoteIdentifier", c("DBIConnection", "SQL"), asNamespace("DBI")))
 
-#' @inherit DBI::dbListTables
-#' @param catalog_name The name of the catalog to return, the default returns all catalogs.
-#' @param schema_name The name of the schema to return, the default returns all schemas.
-#' @param table_name The name of the table to return, the default returns all tables.
+#' List remote tables and fields for an ODBC connection
+#'
+#' `dbListTables()` provides names of remote tables accessible through this
+#' connection; `dbListFields()` provides names of columns within a table.
+#'
+#' @inheritParams DBI::dbListTables
```

Comment:
```suggestion
#' @inheritParams DBI::dbListFields
```

Seeing:

```
❯ checking Rd \usage sections ... WARNING
  Undocumented arguments in documentation object 'dbListTables,OdbcConnection-method'
    'name'
```

in GHA. `?DBI::dbListFields` documents both `conn` and `name` and looks like the place where that `@param`'s docs were inherited from previously.

Reply:
Good catch, thanks!

Reply:
Oh actually, I had this further down; I had just forgotten to re-document 😬